unblob/file_utils.py

```diff
@@ -211,6 +212,8 @@ def __init__(self, file: io.BufferedIOBase, start: int):
         self._file.seek(start)
 
     def seek(self, offset: int, whence=io.SEEK_SET):
+        if (MAX_INT64 <= offset) or (offset <= -(MAX_INT64)):
```

Comment:
As `0` is encoded as "positive" number, the minimum supported value is `-2^63` but the maximum is `2^63-1`